web/packages/plugins/shared/common/src/components/Modal/CustomScrim.tsx

```diff
@@ -1,6 +1,7 @@
 import { Dialog, Icon, Scrim } from '@equinor/eds-core-react'
 import React from 'react'
 import styled from 'styled-components'
+import { isObject } from 'lodash'
```

Comment:
Don't see this being used. Can we remove the "lodash" package?

Reply:
lodash seems to be used indirectly by other packages (not used in any of our package.json files)